core/src/main/java/org/apache/accumulo/core/client/admin/compaction/CompactionSelector.java

```diff
@@ -78,5 +78,5 @@ public Selection(Collection<CompactableFile> filesToCompact) {
     }
   }
 
-  Selection select(SelectionParameters sparams);
```

Comment:
We probably don't want to throw checked exceptions here. The assumption should be that this method doesn't throw any normal exceptions, and that the selector is responsible for handling everything. That way, the only things that can fall through here are RuntimeExceptions.